File: src/main/java/com/sesame/worker/YamlDocumentationWorkerImpl.java
package com.sesame.worker;

import com.sesame.domain.internal.DocMethod;
import org.yaml.snakeyaml.Yaml;

import java.util.Optional;

public class YamlDocumentationWorkerImpl implements DocumentationWorker<Yaml> {

    @Override
    public Optional<DocMethod> processInterfaceMethod(Yaml yaml) {
        return Optional.empty();
    }
}

File: src/main/kotlin/com/sesame/core/worker/YamlDocumentationWorkerImpl.kt
package com.sesame.core.worker

import com.sesame.core.domain.MethodInfo
import org.springframework.stereotype.Component
import java.io.File
import java.util.*

@Component
class YamlDocumentationWorkerImpl : DocumentationWorker {

    override fun processInterfaceMethod(file: File): Optional<MethodInfo> {
        return Optional.empty()
    }
}